app.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const { getTopics } = require("./controllers/app.controllers");
+
+const app = express();
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
+
+app.use((err, req, res, next) => {
+  if ((err.status = "22P02")) {
+    res.status(400).send({ msg: "Invalid Input" });
+  } else {
+    next(err);
+  }
+});
```

Comment:
What's the purpose of this error handler?

Looks to me like you're setting it up in preparation for the future, this isn't a bad idea for the internal error handler below as it's something you won't be testing for but that's not the case for this one. 

You don't have any tests that cover this so you don't know whether it works or not. For this reason I'd take this out as it might be broken and you won't know about it until something goes wrong in the future. This is how bugs get introduced, better to take it out and only commit it once you have some tests that require it and you can be sure that it works the way you want.